v2/weaviate/graphql/nearimagebuilder.go

```diff
@@ -0,0 +1,63 @@
+package graphql
+
+import (
+	"encoding/base64"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"strings"
+)
+
+type NearImageArgumentBuilder struct {
+	image         string
+	fileReader    io.Reader
+	withCertainty bool
+	certainty     float32
+}
+
+// WithImage base64 encoded image
+func (b *NearImageArgumentBuilder) WithImage(image string) *NearImageArgumentBuilder {
+	b.image = image
+	return b
+}
+
+// WithFileReader the image file
+func (b *NearImageArgumentBuilder) WithFileReader(fileReader io.Reader) *NearImageArgumentBuilder {
```

Comment:
I think we can give this one a more generic name, since would also work different readers, such as `bytes.Reader` or basically anything with a `Read([]byte) (int,error)` method.

How about just `.WithReader()`?

Reply:
sounds good!